src/Exthost/ShellLaunchConfig.re

```diff
@@ -1,14 +1,40 @@
+open Oni_Core;
+
+type environment =
+  | Inherit
+  | Additive(StringMap.t(string))
+  | Strict(StringMap.t(string));
+
```

Comment:
These parts of the diff don't look related to the documentation change - I believe it's just a rebase issue. Rebasing to the latest master commit should remove these from the diff view here.

Reply:
Yep you're right, no problem will do a rebase soon and let you know when it's done.
Thanks for the review !

Reply:
Awesome, thank you @AkechiShiro!

Reply:
Rebase done, I think you can safely merge this PR now.

Reply:
Actually @Dark-Matter7232 which has some latest commits merged in master did not sign the CLA it seems so until he signs it I'm not sure merging is possible.
I'll let you handle that, if another rebase is needed in the future I will do them as soon as I can.

Reply:
Thanks @AkechiShiro for your help! Looks like those extra commits were being picked up inadvertently - I revised the history a bit. Bringing it in now :+1: